java/src/jmri/jmrix/loconet/locoio/Bundle.java

```diff
@@ -12,7 +12,7 @@
 
 @javax.annotation.concurrent.Immutable
 
-/**
+/*
```

Comment:
Is this change deliberate? Just checking.

Reply:
yes, flagged as dangling jdoc (class jdoc not to be marked by 2 **)

Reply:
Can you explain this? I want to learn and I don't understand. Should this comment not be a javadoc comment?